app/experimenter/static/rapid/contexts/experiment/context.ts

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import { ExperimentContext } from "experimenter-types/experiment";
+
+export const INITIAL_CONTEXT: ExperimentContext = {
+  state: {
+    name: "",
+    objectives: "",
+  },
+  dispatch(action) {
```

Comment:
What's the purpose of this?

Reply:
i needed something to have this match context but maybe there's a better way to deal with this?

Reply:
hmm yeah ok. i dont know off the top of my head but maybe we can revisit